docs/docs/userguide/configuration/CONFIGURATION.md

```diff
@@ -158,6 +158,14 @@ max_cluster_capacity = 5  # between 1 and 20
     is installed in the Kubernetes cluster. The number of nodes will be automatically adjusted depending on the workload
     resource requirements.
 
+### Logging S3 bucket name
+
+This optional property defines the name of the S3 bucket that will be used to store the logs for the terraform activities.
```

Comment:
I find this a bit misleading in terms of `terraform activities`. The logs that are going to be stored in this bucket are about all actions taken on s3 resources of another bucket, that one happens to store tfstate in our case.